int32 program_counter
int32 macro_number
